cypress/e2e/yves/ssp-service/ssp-service.cy.ts

```diff
@@ -0,0 +1,178 @@
+import {container} from '@utils';
+import {CustomerLoginScenario, CustomerLogoutScenario, CheckoutScenario} from '@scenarios/yves';
+import {SspServiceListPage, CatalogPage, ProductPage} from '@pages/yves';
+
+(['suite'].includes(Cypress.env('repositoryId')) ? describe : describe.skip)(
+    'SSP Service Management',
+    {tags: ['@yves', '@ssp-service', '@ssp', '@SspServiceManagement']},
+    (): void => {
+        const customerLoginScenario = container.get(CustomerLoginScenario);
+        const customerLogoutScenario = container.get(CustomerLogoutScenario);
+        const sspServiceListPage = container.get(SspServiceListPage);
+        const checkoutScenario = container.get(CheckoutScenario);
+
+        let staticFixtures: Record<string, any>;
+        let dynamicFixtures: Record<string, any>;
+        let isSetupDone = false;
+
+        before((): void => {
+            ({staticFixtures, dynamicFixtures} = Cypress.env());
+        });
+
+        // Setup method to be called in each test
+        const prepareServices = () => {
+            if (!isSetupDone) {
+                checkoutScenario.execute({idCustomerAddress: dynamicFixtures.address1.id_customer_address});
+
+                isSetupDone = true;
+            }
+
+            sspServiceListPage.visit();
+        };
+
+        afterEach(() => {
+            customerLogoutScenario.execute();
+        });
+
+        describe('Service List Page', () => {
+            it('should display the service list page with table', (): void => {
+                customerLoginScenario.execute({
+                    email: dynamicFixtures.customer.email,
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                });
+                prepareServices();
+
+                // Assert page is loaded correctly
+                cy.get('h1').should('contain', 'Services');
+                sspServiceListPage.getTable().should('exist');
+            });
+
+            it('should verify all required table headers exist', (): void => {
```

Comment:
It makes sense to merge this test with the previous one. 
In other words, move assertions from this test (table columns check) to the previous one and delete this test.